test/edu/csus/ecs/pc2/core/util/AbstractTestCase.java

```diff
@@ -1566,4 +1567,11 @@ public void assertNoPermssion(edu.csus.ecs.pc2.core.security.Permission.Type typ
         assertFalse("Expected for  " + account + " Permission " + type, account.isAllowed(type));
     }
     
+    public void ensureStaticLog() {
+        
+        if (StaticLog.getLog() == null) {
+            String testDirectory = getOutputDataDirectory();
+            StaticLog.setLog(new Log(testDirectory, "testPrintreport.log"));
```

Comment:
And as noted in the SampleContest test, `testPrintreport.log` is not a good name for a log file which likely has nothing to do with printing reports.  I suggest fixing both the above issues (removing method `ensureStaticLog()` from the SampleContest test, and changing the log here to be placed in the "logs" directory instead of the output data directory), and then using something like "defaultStaticLog.log" for the default log file name.

Reply:
out of scope.

Reply:
You are saying that correcting an improperly-named log file in code that you added to the PR, is out of scope.  I do NOT understand that logic AT ALL....  it's like you're saying "it's ok to add bad code".

Reply:
pushed fix, now pc2 uses LogUtilities ensureStaticLog method